src/kanata/linux.rs

```diff
@@ -84,8 +90,8 @@ impl Kanata {
                     };
                 }
 
-                // Send key events to the processing loop
-                if let Err(e) = tx.try_send(key_event) {
+                // Send key events to the preprocessing loop instead of directly to the processing loop
+                if let Err(e) = preprocess_tx.try_send(key_event) {
```

Comment:
This should send to `tx` if debounce duration is `0` so that users who do not opt in have minimal performance change.

Notably I should do this in the Windows code as well when there is no special altgr processing.